### DISCUSSION TITLE
Xtouch mini & export as layer in photoshop

### DISCUSSION THREAD

**Discussion by FedeCang** [rsjaffe/MIDI2LR#783] [General]
Hi everyone,
is there a way to map a button on Behrigher Mini X to open some photos in Photoshop via the "edit as layer in Photoshop" menu? I used to do this with a macro on my Redragon M908 mouse under Win11, but now I've switched to Mac and it doesn't work anymore.
Thank you in advance for your reply!
Federico.

**Comment by amiramir2124** [comment]
https://khanephotoshop.com/

**Comment by amiramir2124** [comment]
this site can helps you
